wiki/Tournaments/CIC/2020/en.md

```diff
@@ -0,0 +1,173 @@
+
+## Organization
+
+CIC 2020 is run by various osu! community members predominantly hailing from Indonesia.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201) |
+| Map selector | ![][flag_ID] [mirza_rachman18](https://osu.ppy.sh/users/2985335), ![][flag_ID] [-Hex-](https://osu.ppy.sh/users/8630988), ![][flag_ID] [Dapuluous](https://osu.ppy.sh/users/8140944) |
+| Streamer | ![][flag_TW] [XzCraftP](https://osu.ppy.sh/users/1593180), ![][flag_ID] [wiam103](https://osu.ppy.sh/users/4109839), ![][flag_ID] [Victim_Crasher](https://osu.ppy.sh/users/2084869) |
+| Commentator | ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [-Hex-](https://osu.ppy.sh/users/8630988), ![][flag_ID] [Victim_Crasher](https://osu.ppy.sh/users/2084869), ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805), ![][flag_ID] [Shiviaren](https://osu.ppy.sh/users/3429259), ![][flag_ID] [Swirfly](https://osu.ppy.sh/users/5164134), ![][flag_ID] [Feeqzo](https://osu.ppy.sh/users/16779488), ![][flag_ID] [Juragan_Piscok1](https://osu.ppy.sh/users/15507280) |
+| Referee | ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [-Hex-](https://osu.ppy.sh/users/8630988), ![][flag_ID] [mina-](https://osu.ppy.sh/users/4670692), ![][flag_ID] [Avoya](https://osu.ppy.sh/users/3136817), ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805), ![][flag_ID] [Yuina Hotaru](https://osu.ppy.sh/users/1104256),  ![][flag_ID] [- Ruu -](https://osu.ppy.sh/users/3811821), ![][flag_ID] [Koimeji](https://osu.ppy.sh/users/4137039), ![][flag_ID] [Execration-](https://osu.ppy.sh/users/9260926), ![][flag_ID] [Valskiel](https://osu.ppy.sh/users/1254639) |
+| Graphic designer | ![][flag_ID] [mina-](https://osu.ppy.sh/users/4670692), ![][flag_ID] [Avoya](https://osu.ppy.sh/users/3136817) |
+| Spreadsheet manager | ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201) |
+| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |
+
+## Links
+
+- **[Main spreadsheet](http://docs.google.com/spreadsheets/u/3/d/e/2PACX-1vRWcBzTJy5RnGqGIzRDc_Zv28lRo9Z3yz9yUVqF4eqoVi-cebrHDDEWk186ODj71mLf2x9mPARwaVlB/pubhtml?gid=2045538686&single=true)**
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1096883)
+- [Discord server](https://discord.gg/YwAYbPa)
+- [Livestream channel](https://www.twitch.tv/osuIndonesia/)
+
+## Participants
+
+| Group |  |  |  |  |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | TBD | TBD | TBD | TBD |
+| **B** | TBD | TBD | TBD | TBD |
+| **C** | TBD | TBD | TBD | TBD |
+| **D** | TBD | TBD | TBD | TBD |
+| **E** | TBD | TBD | TBD | TBD |
+| **F** | TBD | TBD | TBD | TBD |
+| **G** | TBD | TBD | TBD | TBD |
+| **H** | TBD | TBD | TBD | TBD |
+
```

Comment:
i guess if you are not planning to fill the table in this PR, its better remove the entire header for now 
```suggestion
```

Reply:
i can update it in a bit when the groups are announced tonight i think, the reason i had a template in place was because the screening result hasn't came in yet at the time of writing

Reply:
added participants, all should be good now